Resources/Views/Hub/home.leaf

```diff
@@ -0,0 +1,330 @@
+                <button type="button" class="btn btn-secondary w-100 d-md-none mt-n2 mb-3" data-bs-toggle="collapse" data-bs-target="#ticket-menu">
+                  <i class="bx bxs-user-detail fs-xl me-2"></i>
+                  Ticket Hub Menu
+                  <i class="bx bx-chevron-down fs-lg ms-1"></i>
+                </button>
+                
+                <div id="ticket-menu" class="list-group list-group-flush collapse d-md-block">
+                #if(false):
+                  <a href="#info" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center active">
+                    <i class="bx bx-cog fs-xl opacity-60 me-2"></i>
+                    Personal Information
+                  </a>
+                  <a href="#merch" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bxs-t-shirt fs-xl opacity-60 me-2"></i>
+                    Merch Preferences
+                  </a>
+                #endif
+                
+                #if(count(dropInSessions) > 0):
+                  <a href="#dropin" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bx-chat fs-xl opacity-60 me-2"></i>
+                    Drop-in Sessions
+                  </a>
+                #endif
+                  
+                #if(count(groupSessions) > 0):
+                  <a href="#focus" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bx-group fs-xl opacity-60 me-2"></i>
+                    Focus Groups
+                  </a>
+                #endif
+                  
+                  <a href="#visa" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bxs-traffic-barrier fs-xl opacity-60 me-2"></i>
+                    VISA Invitation
+                  </a>
+                  <a href="#refund" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bx-wallet fs-xl opacity-60 me-2"></i>
+                    Ticket Refunds
+                  </a>
+                  <a href="/ticket/logout" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bx-log-out fs-xl opacity-60 me-2"></i>
+                    Sign Out
+                  </a>
+                </div>
+              </div>
+            </div>
+          </aside>
+
+
+          <!-- Account details -->
+          <div class="col-md-8 offset-lg-1 pb-5 mb-2 mb-lg-4 pt-md-5 mt-n3 mt-md-0">
+            <div class="ps-md-3 ps-lg-0 mt-md-2 py-md-4">
+              <h1 class="h2 pt-xl-1 pb-3">Ticket Details</h1>
+
+#if(false):
```

Comment:
Some chunks are being worked on, and have been disabled for now. Nothing secretive, don't worry about it.